### ISSUE TITLE
Upgrade dependencies

### ISSUE THREAD

**Issue by zbal** [Wiredcraft/api-boilerplate#66]
Plenty of high severity issues in the current repo. 

Need to address them.

<img width="866" alt="Screen Shot 2019-08-06 at 10 20 35" src="https://user-images.githubusercontent.com/631984/62506619-ee28b980-b833-11e9-81b2-bf159899d508.png">

/cc @xavierchow - please assign to whomever can address it.

**Comment by xavierchow** [comment]
Blocked by the merge failure, waiting for @zbal to fix the previous commit msg,
```
~/workspace/api-boilerplate   master  git merge origin/master
husky > npm run -s commitmsg (node v10.14.1)

⧗   input:
Merge remote-tracking branch 'origin/master'

✖   message may not be empty [subject-empty]
✖   type may not be empty [type-empty]
✖   found 2 problems, 0 warnings

husky > commit-msg hook failed (add --no-verify to bypass)
Not committing merge; use 'git commit' to complete the merge.
```

**Comment by xavierchow** [comment]
<img width="702" alt="Screenshot 2019-09-03 21 09 56" src="https://user-images.githubusercontent.com/2748884/64176061-34068b00-ce8f-11e9-94d7-c45db3c2cef2.png">
Better, but still a few alerts. @Zacharysp

**Comment by Zacharysp** [comment]
looks like my yarn is not up to date.

**Comment by xavierchow** [comment]
Much better after merging #75, @Zacharysp do you know why we need swagger-ui in the dependencies?

**Comment by Zacharysp** [comment]
it was there for a while, I guess it was used to create a static api doc web page at the first place?

**Comment by xavierchow** [comment]
@Zacharysp could you try if you can remove it by regenerating the yarn.lock? I don't think it's a mandatory dependency for api-boilerplate.

**Comment by zbal** [comment]
Any update ? @Zacharysp

**Comment by Zacharysp** [comment]
@zbal should been fixed in #71 #75.
